README.md

```diff
@@ -43,43 +42,73 @@ USAGE
   fastly-exporter [flags]
 
 FLAGS
+  -api-refresh 1m0s                        how often to poll api.fastly.com for updated service metadata
+  -api-timeout 15s                         HTTP client timeout for api.fastly.com requests (5–60s)
   -debug false                             Log debug information
   -endpoint http://127.0.0.1:8080/metrics  Prometheus /metrics endpoint
+  -name-exclude-regex ...                  if set, ignore any service whose name matches this regex
+  -name-include-regex ...                  if set, ignore any service whose name doesn't match this regex
```

Comment:
The double negative may make it more confusing than it actually is... What about phrasing like this?
```suggestion
  -name-include-regex ...                  if set, only include services whose name matches this regex
```